src/main/rx/rx.c

```diff
@@ -396,6 +399,15 @@ static void setLinkQuality(bool validFrame, timeDelta_t currentDeltaTime)
     }
 }
 
+void setLinkQualityDirect(uint16_t linkqualityValue)
+{
+#ifndef USE_RX_LINK_QUALITY_INFO
+    UNUSED(linkqualityValue);
```

Comment:
I'd turn this around, and put the desired functionality into the `#if`, and the compiler warning suppression into the `#else` - it will be a bit easier to read then.

Reply:
ok i lost you before you ask me to put  UNUSED(linkqualityValue);  in  #else
i looked and used the same pototype as 
static void updateRSSIADC(timeUs_t currentTimeUs)
{
#ifndef USE_ADC
    UNUSED(currentTimeUs);
#else

so you wish 
#ifdef 
...
 #else 
 UNUSED(currentTimeUs); 
?

Reply:
turn it around